src/commands/gcr-auth.yml

```diff
@@ -44,23 +44,7 @@ steps:
 
   - run:
       name: gcloud auth configure-docker
-      command: |
-        # Set sudo to work whether logged in as root user or non-root user
-        if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
-
-        # configure Docker to use gcloud as a credential helper
-        mkdir -p "$HOME/.docker"
-
-        if [[ "<<parameters.registry-url>>" == *"docker.pkg.dev" ]]; then
-          gcloud auth configure-docker --quiet --project $<<parameters.google-project-id>> <<parameters.registry-url>>
-        else
-          gcloud auth configure-docker --quiet --project $<<parameters.google-project-id>>
-        fi
-
-        # if applicable, provide user access to the docker config file
-        if [[ -d "$HOME/.docker" ]]; then
-          $SUDO chown "$USER:$USER" "$HOME/.docker" -R
-        fi
-        if [[ -d "$HOME/.config" ]]; then
-          $SUDO chown "$USER:$USER" "$HOME/.config" -R
-        fi
+      environment:
+        INDIRECT_PROJECT_ID: << parameters.google-project-id >>
```

Comment:
Environment parameters coming from the orb often have the `PARAM` prefix to separate them from other variables. I would argue that we should go the extra mile and have `ORB_PARAM_` as the standard for all of them. 

Thoughts?

Reply:
Additionally, I like the idea of having a prefix/suffix for `env_var_name` variables, as they need proper expansion. In the unity-orb, I'm suffixing them with `_VAR_NAME` (e.g. https://github.com/EricRibeiro/unity-orb/blob/main/src/commands/prepare-env.yml#L57). Another something to standardize.